src/lib/libraries/sprites.json

```diff
@@ -1,7 +1,6 @@
 [
     {
         "name": "Abby",
-        "baseLayerID": -1,
```

Comment:
Why did all base layer IDs need to be changed to -1? What is baseLayerID used for?

Reply:
This is a change to `libgen` that attempts to normalize all of the assets before the libraries are generated.